placeholderkv.conf

```diff
@@ -360,7 +360,7 @@ logfile ""
 # syslog-enabled no
 
 # Specify the syslog identity.
-# syslog-ident redis
+# syslog-ident placeholderkv
```

Comment:
This isn't aligned with config.c now, should we update that here now too?

Reply:
Addressing it in config.c, would be good to wrap up the file in one go.